### ISSUE TITLE
Use Set-Content not Out-File

### ISSUE THREAD

**Issue by treestryder** [MicrosoftDocs/SCCMdocs#1196]
The [documentation](https://docs.microsoft.com/en-us/sccm/core/get-started/capabilities-in-technical-preview-1810#bkmk_autopilot) should suggest to use Set-Content, rather that Out-File. When using Out-File, the output is passed through the Powershell formatter first, which will truncate long lines and such.

`Get-AutopilotProfile | ConvertTo-AutopilotConfigurationJSON  | Set-Content -Encoding ASCII -Path .\SomePath\SomeFile.josn`

**Comment by aczechowski** [comment]
Thanks @treestryder for the feedback! I hope to incorporate this content into the core docs in the near future, so will make the change at that time.

**Comment by treestryder** [comment]
The following installs the modules, connects, exports all of the Autopilot Profiles in separate files -- with the correct encoding and without formatting...


```
Install-Module AzureAD
Install-Module WindowsAutopilotIntune 
Import-Module WindowsAutopilotIntune 
Connect-AutopilotIntune 

$gap = Get-AutopilotProfile

$gap | %{$_ | ConvertTo-AutoPilotConfigurationJSON | Set-Content -Encoding Ascii "~\desktop\$($_.displayName).json" }

```

**Comment by aczechowski** [comment]
I published a [new article](https://docs.microsoft.com/sccm/osd/deploy-use/windows-autopilot-for-existing-devices) that includes this sample, thanks again for sharing. It will go live tomorrow around 11am Pacific. Thus we closed this issue. 

@treestryder if you're interested in getting contributor status for it, edit the article and make a change. If there's anything on the script samples you would improve, feel free to change. Or merely add a space character anywhere in the article. I'll approve that contribution in this instance. 😃 For more information, see [How to contribute](https://docs.microsoft.com/sccm/core/understand/use-docs#bkmk_contribute).
